plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/partition/HiveHudiPartitionInfo.java

```diff
@@ -69,7 +69,7 @@ public HiveHudiPartitionInfo(
     @Override
     public Table getTable()
     {
-        return null;
+        return table;
```

Comment:
Currently, we don't need this implementation so shall we remove the api from the interface? I believe we will need it for MOR snapshot query support. We can then add it back. What do you think? @geek-will

Reply:
Yes, currently this API is not used and we can remove it. 
I am working on Hudi MOR, we can add it back in the PR of Hudi MOR.

Reply:
@willzgw please remove `io.trino.plugin.hudi.partition.HudiPartitionInfo#getTable` method because it is unused anyway.

Reply:
I suppose the current PR and commit title isn't appropriate. Please update them likes "Remove unused HudiPartitionInfo#getTable". If NPE exactly happens, we need tests.